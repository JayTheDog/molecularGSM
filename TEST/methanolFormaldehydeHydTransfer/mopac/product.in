PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  H -0.110900 -0.043169 0.268626
  C -0.428820 -0.344915 1.269635
  C 0.814568 -0.790167 2.064214
  H -0.926909 0.499130 1.750628
  H 0.968847 -0.152021 2.945149
  O 1.929568 -0.737698 1.173839
  H 2.731941 -0.923461 1.679944
  O 0.530575 -2.547664 3.715411
  C 0.643256 -2.223220 2.555394
  H -1.162827 -1.150010 1.161421
  H 0.644606 -2.973104 1.730409
