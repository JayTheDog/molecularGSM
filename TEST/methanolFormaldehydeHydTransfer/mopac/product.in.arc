

                     SUMMARY OF PM6 CALCULATION, Site No: 14762

                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:53:54 2016
                                                       No. of days left =   0

           Empirical Formula: C3 H6 O2  =    11 atoms

PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -80.24433 KCAL/MOL =    -335.74227 KJ/MOL
          TOTAL ENERGY            =      -1031.77374 EV
          ELECTRONIC ENERGY       =      -3354.21808 EV
          CORE-CORE REPULSION     =       2322.44434 EV
          GRADIENT NORM           =          0.87599
          DIPOLE                  =          1.68956 DEBYE    POINT GROUP:       C1  
          NO. OF FILLED LEVELS    =         15
          IONIZATION POTENTIAL    =         10.026280 EV
          HOMO LUMO ENERGIES (EV) =        -10.026  0.015
          MOLECULAR WEIGHT        =         74.079
          COSMO AREA              =        111.31 SQUARE ANGSTROMS
          COSMO VOLUME            =         96.78 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            O     8    H     1     4.19786
            H     7    H    10     3.92840
            H    11    H     4     3.00435
          SCF CALCULATIONS        =         54
          WALL-CLOCK TIME         =          0.195 SECONDS
          COMPUTATION TIME        =          0.188 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
  H    -0.14137905 +1   0.03166231 +1   0.35416900 +1
  C    -0.40734218 +1  -0.36083374 +1   1.35076717 +1
  C     0.85472775 +1  -0.79892153 +1   2.07898218 +1
  H    -0.93431207 +1   0.43356241 +1   1.89197534 +1
  H     1.12274539 +1  -0.11075022 +1   2.90853297 +1
  O     1.89868616 +1  -0.79901326 +1   1.08622083 +1
  H     2.76805328 +1  -0.63088633 +1   1.49680178 +1
  O     0.28738793 +1  -2.51043539 +1   3.66716685 +1
  C     0.72305727 +1  -2.25086846 +1   2.57166562 +1
  H    -1.12242305 +1  -1.17647251 +1   1.18783267 +1
  H     1.05576564 +1  -3.01316542 +1   1.85094379 +1
 
