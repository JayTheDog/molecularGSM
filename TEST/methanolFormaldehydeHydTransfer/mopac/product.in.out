 *******************************************************************************
 ** Site#: 14762        For non-commercial use only     Version 15.168L 64BITS**
 *******************************************************************************
 ** Cite this work as:                                                        **
 **                                                                           **
 ** (a) MOPAC2012, James J. P. Stewart, Stewart Computational Chemistry,      **
 **  Version 15.168L web: HTTP://OpenMOPAC.net         Days remaining:   0    **
 **                                                                           **
 **                                 AND                                       **
 **                                                                           **
 ** (b) Maia, J.D.C., et al., J. Chem. Theory Comput. 2012, 8, 3072-3081.     **
 **                DOI: 10.1021/ct3004645.                                    **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2012                                  **
 **                                                                           **
 *******************************************************************************

                              PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Fri Dec  2 19:53:54 2016  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  SINGLET    - SPIN STATE DEFINED AS A SINGLET
 *  NOSYM      - POINT-GROUP SYMMETRY SET TO C1
 *  T=         - A TIME OF     10.0 MINUTES REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H         -0.11090000  *  -0.04316900  *   0.26862600  *
     2       C         -0.42882000  *  -0.34491500  *   1.26963500  *
     3       C          0.81456800  *  -0.79016700  *   2.06421400  *
     4       H         -0.92690900  *   0.49913000  *   1.75062800  *
     5       H          0.96884700  *  -0.15202100  *   2.94514900  *
     6       O          1.92956800  *  -0.73769800  *   1.17383900  *
     7       H          2.73194100  *  -0.92346100  *   1.67994400  *
     8       O          0.53057500  *  -2.54766400  *   3.71541100  *
     9       C          0.64325600  *  -2.22322000  *   2.55539400  *
    10       H         -1.16282700  *  -1.15001000  *   1.16142100  *
    11       H          0.64460600  *  -2.97310400  *   1.73040900  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         H       -0.1109   -0.0432    0.2686
     2         C       -0.4288   -0.3449    1.2696
     3         C        0.8146   -0.7902    2.0642
     4         H       -0.9269    0.4991    1.7506
     5         H        0.9688   -0.1520    2.9451
     6         O        1.9296   -0.7377    1.1738
     7         H        2.7319   -0.9235    1.6799
     8         O        0.5306   -2.5477    3.7154
     9         C        0.6433   -2.2232    2.5554
    10         H       -1.1628   -1.1500    1.1614
    11         H        0.6446   -2.9731    1.7304

 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: C3 H6 O2  =    11 atoms



      MOLECULAR POINT GROUP   :   C1  


 SINGLET STATE CALCULATION


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =   15

      Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.016 TIME LEFT: 10.00M  GRAD.:    46.737 HEAT: -78.33406
 CYCLE:     2 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    25.009 HEAT: -79.22770
 CYCLE:     3 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    31.497 HEAT: -79.45495
 CYCLE:     4 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    29.418 HEAT: -79.52397
 CYCLE:     5 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     9.851 HEAT: -79.67093
 CYCLE:     6 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:    10.103 HEAT: -79.71047
 CYCLE:     7 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     6.975 HEAT: -79.73405
 CYCLE:     8 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    10.317 HEAT: -79.75297
 CYCLE:     9 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     6.714 HEAT: -79.78396
 CYCLE:    10 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     8.741 HEAT: -79.80618
 CYCLE:    11 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     4.833 HEAT: -79.83676
 CYCLE:    12 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     4.312 HEAT: -79.87292
 CYCLE:    13 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     6.136 HEAT: -79.90711
 CYCLE:    14 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     4.710 HEAT: -79.93408
 CYCLE:    15 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     3.229 HEAT: -79.94807
 CYCLE:    16 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     3.012 HEAT: -79.95993
 CYCLE:    17 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     4.712 HEAT: -79.97649
 CYCLE:    18 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     3.922 HEAT: -79.99338
 CYCLE:    19 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     2.976 HEAT: -80.01203
 CYCLE:    20 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.467 HEAT: -80.02297
 CYCLE:    21 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     4.659 HEAT: -80.03026
 CYCLE:    22 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     2.450 HEAT: -80.04111
 CYCLE:    23 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     1.273 HEAT: -80.04741
 CYCLE:    24 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.863 HEAT: -80.05485
 CYCLE:    25 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     1.788 HEAT: -80.05962
 CYCLE:    26 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.352 HEAT: -80.06620
 CYCLE:    27 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.432 HEAT: -80.06952
 CYCLE:    28 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.843 HEAT: -80.08236
 CYCLE:    29 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     5.283 HEAT: -80.08255
 CYCLE:    30 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     1.878 HEAT: -80.09382
 CYCLE:    31 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.085 HEAT: -80.09691
 CYCLE:    32 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.636 HEAT: -80.10156
 CYCLE:    33 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     1.823 HEAT: -80.10624
 CYCLE:    34 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.738 HEAT: -80.11529
 CYCLE:    35 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     2.586 HEAT: -80.12540
 CYCLE:    36 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.374 HEAT: -80.13589
 CYCLE:    37 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     5.483 HEAT: -80.15740
 CYCLE:    38 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     3.496 HEAT: -80.17188
 CYCLE:    39 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:    10.296 HEAT: -80.19386
 CYCLE:    40 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.129 HEAT: -80.20971
 CYCLE:    41 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     2.120 HEAT: -80.21859
 CYCLE:    42 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     2.196 HEAT: -80.22568
 CYCLE:    43 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.813 HEAT: -80.23413
 CYCLE:    44 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.096 HEAT: -80.23625
 CYCLE:    45 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     3.792 HEAT: -80.23305
 CYCLE:    46 TIME:   0.000 TIME LEFT: 10.00M  GRAD.:     1.659 HEAT: -80.24285
 CYCLE:    47 TIME:   0.008 TIME LEFT: 10.00M  GRAD.:     0.876 HEAT: -80.24432

     GRADIENT =  0.87599  IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
PM6 singlet NOSYM T=10M
   MOPAC run
   geometry optimization


     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2012 (Version: 15.168L)
                                                       Fri Dec  2 19:53:54 2016
                                                       No. of days left =   0




          FINAL HEAT OF FORMATION =        -80.24433 KCAL/MOL =    -335.74227 KJ/MOL


          TOTAL ENERGY            =      -1031.77374 EV
          ELECTRONIC ENERGY       =      -3354.21808 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =       2322.44434 EV
          COSMO AREA              =        111.31 SQUARE ANGSTROMS
          COSMO VOLUME            =         96.78 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.87599
          IONIZATION POTENTIAL    =         10.026280 EV
          HOMO LUMO ENERGIES (EV) =        -10.026  0.015
          NO. OF FILLED LEVELS    =         15
          MOLECULAR WEIGHT        =         74.079

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            O     8    H     1     4.19786
            H     7    H    10     3.92840
            H    11    H     4     3.00435


          SCF CALCULATIONS        =         54
 
          WALL-CLOCK TIME         =          0.195 SECONDS
          COMPUTATION TIME        =          0.187 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H         -0.14137905  *   0.03166231  *   0.35416900  *
     2       C         -0.40734218  *  -0.36083374  *   1.35076717  *
     3       C          0.85472775  *  -0.79892153  *   2.07898218  *
     4       H         -0.93431207  *   0.43356241  *   1.89197534  *
     5       H          1.12274539  *  -0.11075022  *   2.90853297  *
     6       O          1.89868616  *  -0.79901326  *   1.08622083  *
     7       H          2.76805328  *  -0.63088633  *   1.49680178  *
     8       O          0.28738793  *  -2.51043539  *   3.66716685  *
     9       C          0.72305727  *  -2.25086846  *   2.57166562  *
    10       H         -1.12242305  *  -1.17647251  *   1.18783267  *
    11       H          1.05576564  *  -3.01316542  *   1.85094379  *

                             CARTESIAN COORDINATES

   1    H       -0.141379050     0.031662308     0.354169000
   2    C       -0.407342178    -0.360833736     1.350767170
   3    C        0.854727751    -0.798921528     2.078982179
   4    H       -0.934312073     0.433562411     1.891975342
   5    H        1.122745385    -0.110750221     2.908532969
   6    O        1.898686162    -0.799013258     1.086220826
   7    H        2.768053276    -0.630886334     1.496801780
   8    O        0.287387934    -2.510435391     3.667166852
   9    C        0.723057270    -2.250868457     2.571665615
  10    H       -1.122423048    -1.176472511     1.187832670
  11    H        1.055765640    -3.013165416     1.850943788


           Empirical Formula: C3 H6 O2  =    11 atoms



      MOLECULAR POINT GROUP   :   C1  


                  EIGENVALUES  
 -32.18743 -30.41764 -27.36487 -23.77558 -20.97866 -19.01758 -16.72090 -15.59208
 -15.46142 -14.30563 -13.89964 -13.57840 -12.63817 -11.76272 -10.02628   0.01491
   2.05962   2.67171   3.38122   4.26312   4.53399   4.83577   5.17639   5.33079
   5.46897   6.31286


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          H           0.189268        0.8107     0.81073
    2          C          -0.492487        4.4925     1.08731     3.40517
    3          C           0.066726        3.9333     1.11719     2.81608
    4          H           0.167368        0.8326     0.83263
    5          H           0.135099        0.8649     0.86490
    6          O          -0.551832        6.5518     1.82130     4.73053
    7          H           0.310094        0.6899     0.68991
    8          O          -0.435678        6.4357     1.85173     4.58395
    9          C           0.324700        3.6753     1.16157     2.51373
   10          H           0.165064        0.8349     0.83494
   11          H           0.121679        0.8783     0.87832
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.430     1.130    -1.088     1.627
 HYBRID         0.310     0.070     0.158     0.355
 SUM            0.740     1.201    -0.930     1.690


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  H   0.81073
    2  C   1.08731   1.06394   1.17722   1.16401
    3  C   1.11719   0.86932   1.02716   0.91960
    4  H   0.83263
    5  H   0.86490
    6  O   1.82130   1.30345   1.95458   1.47251
    7  H   0.68991
    8  O   1.85173   1.38112   1.86947   1.33336
    9  C   1.16157   0.67844   0.97889   0.85640
   10  H   0.83494
   11  H   0.87832

          JOB ENDED NORMALLY



 TOTAL JOB TIME:             0.20 SECONDS

 == MOPAC DONE ==
